### ISSUE TITLE
enum34 incompatible with Python 3.6 +

### ISSUE THREAD

**Issue by frederizia** [alxnik/py8583#6]
Hi!

Trying to install `py8583` on Python `3.7` causes problems because `enum34` clashes with the `enum` included in the standard library. It shouldn't really be required for python > 3.4. 

Fix for a [similar issue](https://github.com/smartsheet-platform/smartsheet-python-sdk/pull/123) was this in the `setup.py`:
```
if sys.version_info < (3, 4):
    REQUIRES.append('enum34')
```

Could you add something similar to resolve this?

**Comment by frederizia** [comment]
I tried to create a PR but looks like I don't have access. Changing
```
install_requires=['enum34'],
```
to
```
install_requires=['enum34'] if sys.version_info < (3,4) else [],
```

Should do the trick I believe.

**Comment by alxnik** [comment]
Hi,

Sounds reasonable. I'll try to test it by tomorrow and add it. BTW, you should be able to do a PR, the repo doesn't have any special restrictions.

**Comment by frederizia** [comment]
Thanks for getting back to me so quickly! Not sure why the PR didn't work but I've never done one on a public repo so it's probably my mistake.

**Comment by alxnik** [comment]
fixed on 0f109f5f496ec56c8fb2e671af4fac5e6d80e42f
